docs/configuration/cluster_manager/cluster.rst

```diff
@@ -129,6 +130,11 @@ alt_stat_name
   <config_cluster_manager_cluster_stats>` will be duplicated between the standard statistics and a
   tree specified by this parameter (e.g., *cluster.<alt_stat_name>.*).
 
+dns_refresh_rate_ms
```

Comment:
can you point out here that otherwise (for other cluster types) this setting is just ignored.